Sources/PromotedFirebaseRemoteConfig/ClientConfig+PromotedFirebaseRemoteConfig.swift

```diff
@@ -200,13 +203,10 @@ extension String {
   }
 
   func toCamelCase() -> String {
-    var first = true
     return split(separator: "_")
-      .map { value in
-        let result = first ? value :
-          value.prefix(1).uppercased() + value.dropFirst()
-        first = false
-        return String(result)
+      .enumerated()
+      .map { (index, str) in
```

Comment:
Interesting.  This looks better.  I'm surprised this doesn't break things.

Reply:
Tests still pass. I added a new test to cover an edge case with double underscores.